examples/chart/teleport-cluster/templates/auth/_config.scratch.tpl

```diff
@@ -0,0 +1,13 @@
+{{- define "teleport-cluster.auth.config.scratch" -}}
+{{- required "'auth.teleportConfig' is required in scratch mode" .Values.auth.teleportConfig }}
+proxy_service:
+  enabled: false
+ssh_service:
+  enabled: false
+auth_service:
+  enabled: true
+{{- end -}}
+
+{{- define "teleport-cluster.auth.config.custom" -}}
+{{ fail "'custom' mode has been removed with chart v12 because of the proxy/auth split breaking change, see http://link" }}
```

Comment:
Link still needs to be updated to an actual value here, maybe use a bit.ly redirect or something and update it later?

(I wish goteleport.com had its own redirection service that we used in all error messages tbh)